cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/KafkaCruiseControlServlet.java

```diff
@@ -692,6 +695,9 @@ private boolean getPartitionLoad(HttpServletRequest request, HttpServletResponse
 
     String entriesString = request.getParameter(ENTRIES_PARAM);
     Integer entries = entriesString == null ? Integer.MAX_VALUE : Integer.parseInt(entriesString);
+    String topicToFilter = request.getParameter(TOPIC_PARAM);
```

Comment:
Since these are new parameter added, it'll be good to update `doGet()` documentation to reflect the change in `partition_load` endpoint parameters.

Does the topic partition support regular expressions? I feel it'll be useful to add regular expression support.

Reply:
Thanks for the feedback Abhishek. I am wondering which documentation you are refering to? can you point me to it? I will update it. 

I think the regex support is a good idea. let me add it.

Reply:
I was talking about the comments about the function definition. Looks like you already added that. Thanks!